app/controllers/fields_controller.rb

```diff
@@ -0,0 +1,32 @@
+class FieldsController < AuthenticatedController
+  include FieldParser
+
+  # Returns the form view given a source text
+  def form
+    @form_data = parse_fields(params[:source])
+    @allow_dropdown = JSON.parse(params[:allow_dropdown] || 'false')
```

Comment:
If we are just checking against 'true' or 'false' let's make this a whitelist of string values.

I don't want risking JSON parsing errors by sombebody sending some random stuff via the `:allow_dropdown` parameter.